atlp-rwanda/commanders-rn-medica
# Feature: Payment implementation

**What does this PR do?**

This PR includes the appointment booking  and how the payment methods that will be used in that  process  

**Description of Task to be completed?**

- [x] create appointment table on supabase
- [x] collect data from the app on the appointment screens.

- [x] use flutterwave for the payment options or methods

How should this be manually tested?

Navigate the home screen after logging in then click on one of the doctors under top doctors on the home screen. From there you will be directed to the details of the doctor click on book appointment after you will be directed to a book appointment screen where you will choose the time and date you want an appointment to be later you will be directed to the select package screen after selecting a package and duration of the package you will be directed tho the patient details where a patient will fill his or her problem later  you will see a summary of the appointment you want then you can click on **paywithflutterwave** to pay and  schedule that appointment you will see a sucess message after it you will be directed to the upcoming appointments

Any background context you want to provide?
Here are some of the test cards   https://flutterwavedoc.readme.io/v2.0/docs/test-cards
Check in the appointment table on supabase and see if the appointment you booked is there
What are the relevant pivotal tracker/Trello stories?
N/A

Screenshots (if appropriate)
to test it you can view the following video!

https://github.com/atlp-rwanda/commanders-rn-medica/assets/99283140/dbafb821-763b-4bd9-a45b-b44cc982cd03

Questions:
N/A
